index.html

```diff
@@ -1,239 +1,220 @@
+                          Local Government
+                        </label>
+                      </div>
+                      <div class="checkbox">
+                        <label>
+                          <input type="checkbox" value="" id='cbType4'>
+                          <span class='filter-box filter-yellow'></span>
+                          Tourist Spot
+                        </label>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+
+                <button class='btn btn-primary' id="btnSearch"><i class='fa fa-search'></i> Search</button>
+                <a href="/" class='btn btn-secondary' id="btnReset" title="Reset"><i class="fa fa-repeat" aria-hidden="true"></i> Reset</a>
+                <button class='btn btn-dark' id='btnViewMode' title='Change view'><i class='fa fa-list'></i> List view</button>
+
+              </div>
+            </div>
+          </div>
+
+          <div class='col-md-8'>
+            <div id='mapCanvas' style="display: block; height: 1000px;"></div>
+            <div id="listCanvas" style="display: none">
+              <h2 class='list-header hidden-xs'>Search results <small id='list-result-count'></small></h2>
+              
+              <div class="table-responsive">
+                <table class="table">
+                  <thead>
+                    <tr>
+                      <th scope="col"></th>
+                      <th scope="col">Name</th>
+                      <th scope="col">Address</th>
+                      <th scope="col">Contact</th>
+                      <th scope="col">Description</th>
+                    </tr>
+                  </thead>
+                  <tbody id='results-list'></tbody>
+                </table>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="modal fade" id="modal-pop" tabindex="-1" role="dialog">
+          <div class="modal-dialog modal-lg">
+            <div class="modal-content">
+              <div class="modal-body">
+                <div class="container-fluid bd-example-row">
+                  <div class="row">
+                    <div class="col-md-12">
+                      <button type="button" class="close close-btn" data-dismiss="modal" aria-label="Close"><span aria-hidden="true"><i class="fa fa-times-circle" aria-hidden="true"></i></span></button>
+                      <div id="modal-main"></div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div><!-- /.modal-content -->
+          </div><!-- /.modal-dialog -->
+        </div><!-- /.modal -->
+
+        <style id="maps-styles">
+          #my-map{
+            marker-allow-overlap: true;
+            marker-width:20;
+          }
+
+          #my-map[type_id = 1] { marker-file: url(https://cdn.rawgit.com/pointhi/leaflet-color-markers/master/img/marker-icon-green.png); }
```

Comment:
thats a good point. I removed the local markers since they need to be hosted in a path that is a fully qualified URL.

I agree its a bummer with not being able to load high res markers. This seems to be a limitation of cartocss